packages/bpk-component-breadcrumb/src/__snapshots__/BpkBreadcrumbItem-test.js.snap

```diff
@@ -68,7 +67,6 @@ exports[`BpkBreadcrumbItem should render correctly with a custom class name 1`]
         "width": "1.5rem",
       }
     }
-    viewBox="0 0 24 24"
```

Comment:
I don't really know what viewBox does, but I think removing this will affect the default / max width and height... That might be enforced through our styling anyway 🤷‍♂